src/main/java/org/folio/innreach/domain/listener/KafkaInventoryEventListener.java

```diff
@@ -43,8 +43,11 @@ public void handleItemEvents(List<ConsumerRecord<String, DomainEvent<Item>>> con
     log.info("Handling inventory item events from Kafka [number of events: {}]", consumerRecords.size());
 
     var events = getEvents(consumerRecords);
-
+    for(DomainEvent<Item> event : events){
+      log.info("handleItemEvents:: Event type: {}, tenant: {}, timestamp: {}, data: {} ", event.getType(), event.getTenant(), event.getTimestamp(), event.getData());
```

Comment:
event.getData cant give me the ID'd. You will have to print 2 ID's the new and the old one I suppose. But make sure there is no NPE. Make event.getData().getNewEntity().getID() and also  event.getData().getOldEntity().getID().